server/src/main/java/org/elasticsearch/index/stats/IndexingPressureStats.java

```diff
@@ -146,6 +155,10 @@ public void writeTo(StreamOutput out) throws IOException {
         if (out.getTransportVersion().onOrAfter(TransportVersions.INDEXING_PRESSURE_DOCUMENT_REJECTIONS_COUNT)) {
             out.writeVLong(primaryDocumentRejections);
         }
+
+        if (out.getTransportVersion().onOrAfter(TransportVersions.INDEXING_PRESSURE_REQUEST_REJECTIONS_COUNT)) {
```

Comment:
I believe we need a new transport version constant here instead.

Reply:
why? constant name is different from line 155. should be ok, right?